### ISSUE TITLE
[BUG] printf invalid number error on some locale

### ISSUE THREAD

**Issue by Marcin648** [GameServerManagers/LinuxGSM#3587]
## User Story

If user use any command that use `info_distro` they get printf invalid number error.

## Basic info

* **Distro:** [Ubuntu 20.04] (any distro)
* **LinuxGSM version:** [v20.1.3]

## Further Information

```
linuxgsm@LGSM:~/srv$ locale
LANG=pl_PL.UTF-8
LANGUAGE=pl
LC_CTYPE="pl_PL.UTF-8"
LC_NUMERIC="pl_PL.UTF-8"
LC_TIME="pl_PL.UTF-8"
LC_COLLATE="pl_PL.UTF-8"
LC_MONETARY="pl_PL.UTF-8"
LC_MESSAGES="pl_PL.UTF-8"
LC_PAPER="pl_PL.UTF-8"
LC_NAME="pl_PL.UTF-8"
LC_ADDRESS="pl_PL.UTF-8"
LC_TELEPHONE="pl_PL.UTF-8"
LC_MEASUREMENT="pl_PL.UTF-8"
LC_IDENTIFICATION="pl_PL.UTF-8"
LC_ALL=
linuxgsm@LGSM:~/srv$ ./dayzserver start
/home/linuxgsm/srv/lgsm/functions/info_distro.sh: wiersz 39: printf: 20.04: nieprawidłowa liczba
/home/linuxgsm/srv/lgsm/functions/info_distro.sh: wiersz 39: printf: 20.04: nieprawidłowa liczba
/home/linuxgsm/srv/lgsm/functions/info_distro.sh: wiersz 39: printf: 20.04: nieprawidłowa liczba
[  OK  ] Starting dayzserver: LinuxGSM

core_exit.sh exiting with code: 0

```
TL;DR. info_distro used printf to cut off decimal of distro version. Some countries use comma (,) instead of dot (.) for number separation and printf adjusts to that. This problem do not exist when we use standard `en_US.UTF-8`

few printf examples:
```
linuxgsm@LGSM:~/srv$ LC_NUMERIC="en_US.utf8" printf "%.0f\n" 20.04
20
```
```
linuxgsm@LGSM:~/srv$ LC_NUMERIC="pl_PL.utf8" printf "%.0f\n" 20.04
bash: printf: 20.04: nieprawidłowa liczba
20
```

## To Reproduce

Steps to reproduce the behaviour:
1. Fresh install ubuntu server 20.04
2. Use locale that use comma as number separator. (for example Polish `pl_PL.UTF-8`)
3. Use LGSM install/start command on any game.
4. See error `info_distro.sh: wiersz 39: printf: 20.04: nieprawidłowa liczba`

**Comment by dgibbs64** [comment]
Not sure the best solution to this yet. It looks like we still get the correct output. so maybe just silencing the error would do it

**Comment by Marcin648** [comment]
I don't know if it would be better not to use printf.
The easiest way can be use `LC_NUMERIC=C` before command line.
```
$ printf "%.0f\n" "20.04"
20
LC_NUMERIC=C printf "%.0f\n" "20.046"
```
... but printf have some more trap. 🙃 
```
$ LC_NUMERIC=C printf "%.0f\n" "3.5"
4
```
```
~ $ LC_NUMERIC=C printf "%.0f\n" "10.5"
10
```
```
~ $ LC_NUMERIC=C printf "%.0f\n" "10.6"
11
```

May by regex is better?
```
sed -nr 's/^VERSION_ID="([0-9]*).+?"/\1/p' /etc/os-release
```
